doc/plugin_server_nodeattestor_gcp_iit.md

```diff
@@ -39,6 +42,23 @@ If `use_instance_metadata` is true, then the Google Compute Engine API is querie
 | -------------------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------------|
 | `gcp_iit:tag`              | `gcp_iit:tag:blog-server`                                    | Instance tag (one selector per)
 | `gcp_iit:sa`               | `gcp_iit:sa:123456789-compute@developer.gserviceaccount.com` | Service account (one selector per) 
+| `gcp_iit:label`            | `gcp_iit:label:key:value`                                    | Instance label
+| `gcp_iit:metadata`         | `gcp_iit:metadata:key:value`                                 | Instance metadata (see caveat below)
+
+Not all instance label and metadata values are useful for node selection. To
+prevent the creation of large amounts of useless selectors, labels and metadata
+are not used by default. To opt-in to use a specific label or metadata value,
+specify the key in the `allowed_label_keys` or `allowed_metadata_keys`
```

Comment:
Should we note that in order for `allowed_metadata_keys` to work, the `use_instance_metadata` config flag must also be set?

Reply:
It's hard to see in the diff, but the selector table is split into those provided by the IIT and those provided by the instance metadata, separated by the following:

> If `use_instance_metadata` is true, then the Google Compute Engine API is queried for instance metadata which is used to populate these additional selectors: